tutorials/data-science-preprocessing/index.md

```diff
@@ -325,28 +325,34 @@ some setup:
 
 * First, make sure you're using the correct project:
 
-      $ gcloud config set project your-project-id
-      Updated property [core/project]
-      $
+    ```
```

Comment:
Can't use triple backticks in lists, it won't render properly. It looks like this code blocks just needed to be indented an extra 4 spaces.